driver/docker-container/factory.go

```diff
@@ -49,6 +50,20 @@ func (f *factory) New(ctx context.Context, cfg driver.InitConfig) (driver.Driver
 			}
 		case k == "image":
 			d.image = v
+		case k == "memory":
+			d.memory = v
+		case k == "memory-swap":
+			d.memorySwap = v
+		case k == "cpu-period":
+			d.cpuPeriod = int64(cast.ToInt(v))
```

Comment:
I think `strconv.ParseInt` makes more sense here and then we don't need a dependency. The `container create` command uses the cli flags and that uses `strconv.ParseInt` internally.

Reply:
@jsternberg done